eng/Signing.props

```diff
@@ -1,7 +1,7 @@
 ﻿<Project>
   <ItemGroup>
-    <FileSignInfo Include="Microsoft.DiaSymReader.dll" PublicKeyToken="31bf3856ad364e35" TargetFramework=".NETFramework,Version=v2.0" CertificateName="MicrosoftWin8WinBlue"/>
-    <FileSignInfo Include="Microsoft.DiaSymReader.dll" PublicKeyToken="31bf3856ad364e35" TargetFramework=".NETStandard,Version=v1.1" CertificateName="Microsoft101240624"/>
```

Comment:
What to do with these metadata? They are outdated.

Reply:
This is used to target the certificate to a specific TFM. You can't have two entries that have the same file name and PKT.

Reply:
Reverted changes.